.github/workflows/dispatch_opm_simulators.yml

```diff
@@ -0,0 +1,29 @@
+name: Dispatch to opm-simulators
+
+on:
+  push:
+    branches: master
+    paths:
+      - 'python/docstrings_common.json'
+  pull_request:
+    branches: master
+    paths:
+      - 'python/docstrings_common.json'
+
+jobs:
+  dispatch:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Send dispatch to opm-simulators
+      env:
+        PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+      run: |
+        curl -X POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "Authorization: token ${{ secrets.PERSONAL_ACCESS_TOKEN }}" \
+          https://api.github.com/repos/lisajulia/opm-simulators/dispatches \
```

Comment:
I guess this needs to change to `OPM` instead of `lisajulia` when this workflow is activated ?

Reply:
Yes, thanks! I've opened a new PR: https://github.com/OPM/opm-common/pull/4129